### PR TITLE
Add Estimote nearable support

### PR DESCRIPTION
**Pull request by ShaunPlummer**
Add packet parsing and scanning support for Estimote 'Sticker Beacons' advertising 'Nearable' frames.

### CODE REVIEW

File: beacontools/scanner.py [hunk 1]
```diff
@@ -168,6 +168,7 @@ def process_packet(self, pkt):
         if not ( \
             ((self.mode & ScannerMode.MODE_IBEACON) and (pkt[19:23] == b"\x4c\x00\x02\x15")) or \
             ((self.mode & ScannerMode.MODE_EDDYSTONE) and (pkt[19:21] == b"\xaa\xfe")) or \
+            ((self.mode & ScannerMode.MODE_ESTIMOTE) and (pkt[23:25] == b"\x5d\x01")) or \
```
Comment (ShaunPlummer) [inline beacontools/scanner.py]: I wasn't sure if reusing the same device filter and scan mode setting would be acceptable or not.

**Comment by ShaunPlummer** [comment]
What's the coverage threshold for the project?

**Comment by citruz** [comment]
Thank you for your contribution!
I have one comment regarding your code. You added another function call to `parse_packet`. However, since the Estimote Nearable packet is also using a length-tag-value structure. Please adapt your code to use the `parse_ltv_packet` method. In the loop over the frames, you can check for the manufacturer specific data tag (0xff) and compare the first to bytes to the `ESTIMOTE_MANUFACTURER_BYTES` constant.

**Comment by ShaunPlummer** [comment]
I've updated the PR to iterate over the advert data elements and check for a manufacturer specific type, moving the LTV struct into the parser. I also believe this could stand to be more generic.

**Comment by citruz** [comment]
Great, thanks! I merged your changes into a release branch where I will integrate this PR and #36.

**Comment by citruz** [comment]
I merged #35 and #36 and refactored all beacon definitions to use the generic ltv parser. The tests look fine, but since I don't have the hardware at hand, I cannot test the actual scanner.
Please verify that I didn't break anything during refactoring by checking out the release1.4.0 branch. As soon as you confirm that everything works as expected, I will finalize the release.
Thanks!

**Comment by ShaunPlummer** [comment]
Apologies for the delay in replying. I've tested the release1.4.0 branch and everything is working.

**Pull request merged**